x-pack/plugins/screenshotting/server/browsers/download/index.test.ts

```diff
@@ -88,11 +88,11 @@ describe('ensureDownloaded', () => {
         expect.arrayContaining([
           'chrome-mac.zip',
           'chrome-win.zip',
-          'chromium-749e738-locales-linux_x64.zip',
+          'chromium-7abd50c-locales-linux_x64.zip',
         ])
       );
       expect(readdirSync(path.resolve(`${paths.archivesPath}/arm64`))).toEqual(
-        expect.arrayContaining(['chrome-mac.zip', 'chromium-749e738-locales-linux_arm64.zip'])
+        expect.arrayContaining(['chrome-mac.zip', 'chromium-7abd50c-locales-linux_arm64.zip'])
```

Comment:
We should probably make this test less strict, so that it doesn't need to be updated each time. WDYT?

Reply:
improved! https://github.com/elastic/kibana/pull/153033/commits/73549eb069f89efb9232eb2d1d72bfd14fbafb9e